### ISSUE TITLE
[Error] Could not find the image either in docker hub or locally: docker.io/kabanero/nodejs-express:0.2

### ISSUE THREAD

**Issue by karimbzu** [kabanero-io/kabanero-foundation#160]
Kindly help me to resolve the subject issue. I am using openshift 4.2.16 with kabanero 0.5. Trying to deploy nodejs-express using github webhooks. 
![image](https://user-images.githubusercontent.com/34478122/74105139-2cfa4c00-4b96-11ea-9c6e-78d5777bc481.png)
Thanks

**Comment by kvijai82** [comment]
Could you please double check you have the correct credentials specified in your dockerhub secret?  Have seen this fail if that was not right.

**Comment by karimbzu** [comment]
I am using local openshift registry to push image to i.e image-registry.openshift-image-registry.svc:5000, do i also need to specify credentials in a secret for the local openshift registry before during webhook creation?

**Comment by karimbzu** [comment]
i think the command which is causing the error is: "buildah pull docker.io/kabanero/java-microprofile:0.2", it should be docker pull

**Comment by kvijai82** [comment]
Docker is not available in the containers/openshift.  So the pipelines use
a tool called buildah instead.  Also you should not need to specify
dockerhub credentials if you are pushing to the internal registry.  Could
you please walk me through the steps you followed to setup the webhook?

**Comment by karimbzu** [comment]
Thanks for your reply. Please see below the webhook settings:
![image](https://user-images.githubusercontent.com/34478122/74445860-0435c780-4eb2-11ea-8266-6905e6c4311b.png)

**Comment by kvijai82** [comment]
Could you please change the `service account` to `kabanero-pipelines`?

**Comment by karimbzu** [comment]
I dont know what happened with my configuration, suddenly i am unable to create webhook, last time i did: 
![image](https://user-images.githubusercontent.com/34478122/74450330-e0c24b00-4eb8-11ea-972b-792189ff047f.png)

**Comment by karimbzu** [comment]
able to create webhook using "kabanero-pipeline" service account, however, it does not trigger event from github. Similarly, if i create webhook using "kabanero-operator" service account, then trigger initiated a new pipelinerun, however the same build issue appears.

**Comment by karimbzu** [comment]
#oc logs -f el-tekton-webhooks-eventlistener-6b65b6b7cd-7ws7z
*2020/02/13 16:08:08 Error Intercepting Event (EventID: kqzx4): "Not OK response from Event Processor: Request rejected; status: 417 Expectation Failed; message: <nil>\n"
2020/02/13 16:08:08 Error Intercepting Event (EventID: kqzx4): "Not OK response from Event Processor: Request rejected; status: 417 Expectation Failed; message: <nil>\n"
2020/02/13 16:08:08 Error Intercepting Event (EventID: kqzx4): "Not OK response from Event Processor: Request rejected; status: 417 Expectation Failed; message: <nil>\n"
2020/02/13 16:09:35 EventListener: tekton-webhooks-eventlistener in Namespace: tekton-pipelines handling event (EventID: s9f8l) with payload: {"ref":"refs/heads/master","before":"1647052c6b2b42298aa7a88d81d765d240b9e8dd","after":"e16ff52d372953976f87e9f699f989b54da8a207","repository":{"id":228409715,"node_id":"MDEwOlJlcG9zaXRvcnkyMjg0MDk3MTU=","name":"kabanero-java-microprofile","full_name":"karimbzu/kabanero-java-microprofile","private":false,"owner":{"name":"karimbzu","email":"34478122+karimbzu@users.noreply.github.com","login":"karimbzu","id":34478122,"node_id":"MDQ6VXNlcjM0NDc4MTIy","avatar_url":"https://avatars3.githubusercontent.com/u/34478122?v=4","gravatar_id":"","url":"https://api.github.com/users/karimbzu","html_url":"https://github.com/karimbzu","followers_url":"https://api.github.com/users/karimbzu/followers","following_url":"https://api.github.com/users/karimbzu/following{/other_user}","gists_url":"https://api.github.com/users/karimbzu/gists{/gist_id}","starred_url":"https://api.github.com/users/karimbzu/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/karimbzu/subscriptions","organizations_url":"https://api.github.com/users/karimbzu/orgs","repos_url":"https://api.github.com/users/karimbzu/repos","events_url":"https://api.github.com/users/karimbzu/events{/privacy}","received_events_url":"https://api.github.com/users/karimbzu/received_events","type":"User","site_admin":false},"html_url":"https://github.com/karimbzu/kabanero-java-microprofile","description":null,"fork":false,"url":"https://github.com/karimbzu/kabanero-java-microprofile","forks_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/forks","keys_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/keys{/key_id}","collaborators_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/collaborators{/collaborator}","teams_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/teams","hooks_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/hooks","issue_events_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/issues/events{/number}","events_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/events","assignees_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/assignees{/user}","branches_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/branches{/branch}","tags_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/tags","blobs_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/git/blobs{/sha}","git_tags_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/git/tags{/sha}","git_refs_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/git/refs{/sha}","trees_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/git/trees{/sha}","statuses_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/statuses/{sha}","languages_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/languages","stargazers_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/stargazers","contributors_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/contributors","subscribers_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/subscribers","subscription_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/subscription","commits_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/commits{/sha}","git_commits_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/git/commits{/sha}","comments_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/comments{/number}","issue_comment_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/issues/comments{/number}","contents_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/contents/{+path}","compare_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/compare/{base}...{head}","merges_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/merges","archive_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/{archive_format}{/ref}","downloads_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/downloads","issues_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/issues{/number}","pulls_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/pulls{/number}","milestones_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/milestones{/number}","notifications_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/notifications{?since,all,participating}","labels_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/labels{/name}","releases_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/releases{/id}","deployments_url":"https://api.github.com/repos/karimbzu/kabanero-java-microprofile/deployments","created_at":1576507828,"updated_at":"2020-02-13T16:03:33Z","pushed_at":1581610173,"git_url":"git://github.com/karimbzu/kabanero-java-microprofile.git","ssh_url":"git@github.com:karimbzu/kabanero-java-microprofile.git","clone_url":"https://github.com/karimbzu/kabanero-java-microprofile.git","svn_url":"https://github.com/karimbzu/kabanero-java-microprofile","homepage":null,"size":27,"stargazers_count":0,"watchers_count":0,"language":"Java","has_issues":true,"has_projects":true,"has_downloads":true,"has_wiki":true,"has_pages":false,"forks_count":0,"mirror_url":null,"archived":false,"disabled":false,"open_issues_count":0,"license":null,"forks":0,"open_issues":0,"watchers":0,"default_branch":"master","stargazers":0,"master_branch":"master"},"pusher":{"name":"karimbzu","email":"34478122+karimbzu@users.noreply.github.com"},"sender":{"login":"karimbzu","id":34478122,"node_id":"MDQ6VXNlcjM0NDc4MTIy","avatar_url":"https://avatars3.githubusercontent.com/u/34478122?v=4","gravatar_id":"","url":"https://api.github.com/users/karimbzu","html_url":"https://github.com/karimbzu","followers_url":"https://api.github.com/users/karimbzu/followers","following_url":"https://api.github.com/users/karimbzu/following{/other_user}","gists_url":"https://api.github.com/users/karimbzu/gists{/gist_id}","starred_url":"https://api.github.com/users/karimbzu/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/karimbzu/subscriptions","organizations_url":"https://api.github.com/users/karimbzu/orgs","repos_url":"https://api.github.com/users/karimbzu/repos","events_url":"https://api.github.com/users/karimbzu/events{/privacy}","received_events_url":"https://api.github.com/users/karimbzu/received_events","type":"User","site_admin":false},"created":false,"deleted":false,"forced":false,"base_ref":null,"compare":"https://github.com/karimbzu/kabanero-java-microprofile/compare/1647052c6b2b...e16ff52d3729","commits":[{"id":"e16ff52d372953976f87e9f699f989b54da8a207","tree_id":"24f3190f60411811872c026535fc876aae1a5718","distinct":true,"message":"Update README.md","timestamp":"2020-02-14T00:09:33+08:00","url":"https://github.com/karimbzu/kabanero-java-microprofile/commit/e16ff52d372953976f87e9f699f989b54da8a207","author":{"name":"karimbzu","email":"34478122+karimbzu@users.noreply.github.com","username":"karimbzu"},"committer":{"name":"GitHub","email":"noreply@github.com","username":"web-flow"},"added":[],"removed":[],"modified":["README.md"]}],"head_commit":{"id":"e16ff52d372953976f87e9f699f989b54da8a207","tree_id":"24f3190f60411811872c026535fc876aae1a5718","distinct":true,"message":"Update README.md","timestamp":"2020-02-14T00:09:33+08:00","url":"https://github.com/karimbzu/kabanero-java-microprofile/commit/e16ff52d372953976f87e9f699f989b54da8a207","author":{"name":"karimbzu","email":"34478122+karimbzu@users.noreply.github.com","username":"karimbzu"},"committer":{"name":"GitHub","email":"noreply@github.com","username":"web-flow"},"added":[],"removed":[],"modified":["README.md"]}} and header: map[X-Github-Delivery:[3983ef20-4e7b-11ea-8930-bc0a9ef63f85] X-Hub-Signature:[sha1=c7e1154d277d8eef23c76ac0cce529537f9c505f] Content-Length:[7974] X-Forwarded-Host:[el-tekton-webhooks-eventlistener-tekton-pipelines.apps.ocp.tmrnd.net] X-Forwarded-Port:[443] X-Forwarded-Proto:[https] Accept:[*/*] X-Github-Event:[push] Content-Type:[application/json] Forwarded:[for=192.168.50.9;host=el-tekton-webhooks-eventlistener-tekton-pipelines.apps.ocp.tmrnd.net;proto=https;proto-version=""] X-Forwarded-For:[192.168.50.9] User-Agent:[GitHub-Hookshot/1936f4e]]
2020/02/13 16:09:35 Error Intercepting Event (EventID: s9f8l): "Not OK response from Event Processor: Request rejected; status: 417 Expectation Failed; message: <nil>\n"
2020/02/13 16:09:35 Error Intercepting Event (EventID: s9f8l): "Not OK response from Event Processor: Request rejected; status: 417 Expectation Failed; message: <nil>\n"
^C
[root@bastion 0.5]# oc get pods
NAME                                                   READY   STATUS    RESTARTS   AGE
el-tekton-webhooks-eventlistener-6b65b6b7cd-7ws7z      1/1     Running   0          2m35s
tekton-dashboard-5f7746c5f9-s49m4                      2/2     Running   0          16m
tekton-webhooks-extension-validator-5d8dbffdc8-rpvcl   1/1     Running   0          16m
webhooks-extension-5f4b97b7c9-6457r                    1/1     Running   0          16m
*

**Comment by kvijai82** [comment]
Either `pipeline` or `kabanero-pipeline` service accounts are recommended
to use with pipelines.  We have not tested with `kabanero-operator` and are
not sure if it works with pipelines.  That one is intended to be used by
the Kabanero operator.
If you are not seeing events in github getting triggered then it might be a
different issue.  Did you delete the other webhook before creating the one
with the `kabanero-pipeline` service account?  Could you also please check
the status of the webhook in your github repo and see if it's sending
events?

**Comment by karimbzu** [comment]
Thanks for your reply. Yes whenever i created new webhook with new service account i always delete the existing one. Similarly, github repo status: 
![image](https://user-images.githubusercontent.com/34478122/74454731-4a455800-4ebf-11ea-8efc-976b3f906ffc.png) and recent delivers: 
![image](https://user-images.githubusercontent.com/34478122/74454847-7365e880-4ebf-11ea-8d7c-73fc283343fa.png)

**Comment by karimbzu** [comment]
Just now i tried "pipeline" service account and building started successfully. But why "kabanero-pipeline" service account does not initiate trigger? Anyways, thanks a lot for your support, effort and time.

**Comment by kvijai82** [comment]
Oh that's interesting.  I use `kabanero-pipeline` to drive webhooks and it works for me.  I requested  someone with a bit more insight into the webhooks to look at the logs you posted above and chime in with their thoughts.  

The main drawback of `pipelines` is you can't deploy into a different namespace.

**Comment by akihikokuroda** [comment]
Can I see the log of the tekton-webhookd-extension-validator pod?

**Comment by karimbzu** [comment]
[root@bastion 0.5]# oc logs -f tekton-webhooks-extension-validator-5d8dbffdc8-rpvcl
2020/02/13 15:54:00 Interceptor started
2020/02/13 15:58:52 [test-kabanero-pullrequest-event] Validation FAIL (error payload signature check failed validating payload)
2020/02/13 15:58:52 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (error payload signature check failed validating payload)
2020/02/13 15:58:52 [test-kabanero-push-event] Validation FAIL (error payload signature check failed validating payload)
2020/02/13 16:00:04 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (error payload signature check failed validating payload)
2020/02/13 16:00:04 [test-kabanero-pullrequest-event] Validation FAIL (error payload signature check failed validating payload)
2020/02/13 16:00:04 [test-kabanero-push-event] Validation FAIL (error payload signature check failed validating payload)
2020/02/13 16:03:04 [test-kabanero-push-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:03:04 [test-kabanero-push-event] Handling GitHub Event with delivery ID: 4fcc7f00-4e7a-11ea-8095-fba0441a680f
2020/02/13 16:03:04 [test-kabanero-push-event] Validation FAIL (event type does not match, got ping but wanted push)
2020/02/13 16:03:04 [github.com/karimbzu/kabanero-java-microprofile] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:03:04 [github.com/karimbzu/kabanero-java-microprofile] Handling GitHub Event with delivery ID: 4fcc7f00-4e7a-11ea-8095-fba0441a680f
2020/02/13 16:03:04 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (event type does not match, got ping but wanted pull_request)
2020/02/13 16:03:04 [test-kabanero-pullrequest-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:03:04 [test-kabanero-pullrequest-event] Handling GitHub Event with delivery ID: 4fcc7f00-4e7a-11ea-8095-fba0441a680f
2020/02/13 16:03:04 [test-kabanero-pullrequest-event] Validation FAIL (event type does not match, got ping but wanted pull_request)
2020/02/13 16:03:33 [test-kabanero-push-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:03:33 [test-kabanero-push-event] Handling GitHub Event with delivery ID: 619f969a-4e7a-11ea-9b8a-9bbda27cbbc6
2020/02/13 16:03:33 [test-kabanero-push-event] Validation PASS (repository URL, secret payload, event type checked)
2020/02/13 16:03:33 [test-kabanero-push-event] Validation PASS so writing response
2020/02/13 16:03:33 [test-kabanero-pullrequest-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:03:33 [test-kabanero-pullrequest-event] Handling GitHub Event with delivery ID: 619f969a-4e7a-11ea-9b8a-9bbda27cbbc6
2020/02/13 16:03:33 [test-kabanero-pullrequest-event] Validation FAIL (event type does not match, got push but wanted pull_request)
2020/02/13 16:03:33 [github.com/karimbzu/kabanero-java-microprofile] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:03:33 [github.com/karimbzu/kabanero-java-microprofile] Handling GitHub Event with delivery ID: 619f969a-4e7a-11ea-9b8a-9bbda27cbbc6
2020/02/13 16:03:33 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (event type does not match, got push but wanted pull_request)
2020/02/13 16:08:08 [test-kabanero-push-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:08:08 [test-kabanero-push-event] Handling GitHub Event with delivery ID: 0597cd80-4e7b-11ea-8a44-b84343942988
2020/02/13 16:08:08 [test-kabanero-push-event] Validation FAIL (event type does not match, got ping but wanted push)
2020/02/13 16:08:08 [github.com/karimbzu/kabanero-java-microprofile] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:08:08 [github.com/karimbzu/kabanero-java-microprofile] Handling GitHub Event with delivery ID: 0597cd80-4e7b-11ea-8a44-b84343942988
2020/02/13 16:08:08 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (event type does not match, got ping but wanted pull_request)
2020/02/13 16:08:08 [test-kabanero-pullrequest-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:08:08 [test-kabanero-pullrequest-event] Handling GitHub Event with delivery ID: 0597cd80-4e7b-11ea-8a44-b84343942988
2020/02/13 16:08:08 [test-kabanero-pullrequest-event] Validation FAIL (event type does not match, got ping but wanted pull_request)
2020/02/13 16:09:35 [test-kabanero-push-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:09:35 [test-kabanero-push-event] Handling GitHub Event with delivery ID: 3983ef20-4e7b-11ea-8930-bc0a9ef63f85
2020/02/13 16:09:35 [test-kabanero-push-event] Validation PASS (repository URL, secret payload, event type checked)
2020/02/13 16:09:35 [test-kabanero-push-event] Validation PASS so writing response
2020/02/13 16:09:35 [test-kabanero-pullrequest-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:09:35 [test-kabanero-pullrequest-event] Handling GitHub Event with delivery ID: 3983ef20-4e7b-11ea-8930-bc0a9ef63f85
2020/02/13 16:09:35 [test-kabanero-pullrequest-event] Validation FAIL (event type does not match, got push but wanted pull_request)
2020/02/13 16:09:35 [github.com/karimbzu/kabanero-java-microprofile] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:09:35 [github.com/karimbzu/kabanero-java-microprofile] Handling GitHub Event with delivery ID: 3983ef20-4e7b-11ea-8930-bc0a9ef63f85
2020/02/13 16:09:35 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (event type does not match, got push but wanted pull_request)
2020/02/13 16:19:50 [test-kabanero-push-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:19:50 [test-kabanero-push-event] Handling GitHub Event with delivery ID: a8048080-4e7c-11ea-9197-340d13f0ff42
2020/02/13 16:19:50 [test-kabanero-push-event] Validation FAIL (event type does not match, got ping but wanted push)
2020/02/13 16:19:50 [test-kabanero-pullrequest-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:19:50 [test-kabanero-pullrequest-event] Handling GitHub Event with delivery ID: a8048080-4e7c-11ea-9197-340d13f0ff42
2020/02/13 16:19:50 [test-kabanero-pullrequest-event] Validation FAIL (event type does not match, got ping but wanted pull_request)
2020/02/13 16:19:50 [github.com/karimbzu/kabanero-java-microprofile] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:19:50 [github.com/karimbzu/kabanero-java-microprofile] Handling GitHub Event with delivery ID: a8048080-4e7c-11ea-9197-340d13f0ff42
2020/02/13 16:19:50 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (event type does not match, got ping but wanted pull_request)
2020/02/13 16:20:19 [test-kabanero-push-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:20:19 [test-kabanero-push-event] Handling GitHub Event with delivery ID: b9555684-4e7c-11ea-84fb-ea2439c3d0ec
2020/02/13 16:20:19 [test-kabanero-push-event] Validation PASS (repository URL, secret payload, event type checked)
2020/02/13 16:20:19 [test-kabanero-pullrequest-event] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:20:19 [test-kabanero-pullrequest-event] Handling GitHub Event with delivery ID: b9555684-4e7c-11ea-84fb-ea2439c3d0ec
2020/02/13 16:20:19 [test-kabanero-pullrequest-event] Validation FAIL (event type does not match, got push but wanted pull_request)
2020/02/13 16:20:19 [test-kabanero-push-event] Validation PASS so writing response
2020/02/13 16:20:19 [github.com/karimbzu/kabanero-java-microprofile] Clone URL coming in as JSON: https://github.com/karimbzu/kabanero-java-microprofile.git
2020/02/13 16:20:19 [github.com/karimbzu/kabanero-java-microprofile] Handling GitHub Event with delivery ID: b9555684-4e7c-11ea-84fb-ea2439c3d0ec
2020/02/13 16:20:19 [github.com/karimbzu/kabanero-java-microprofile] Validation FAIL (event type does not match, got push but wanted pull_request)

**Comment by karimbzu** [comment]
again tried with "kabanero-pipeline" and failed: 
![image](https://user-images.githubusercontent.com/34478122/74457411-2be15b80-4ec3-11ea-89eb-7cb557750ebb.png)

**Comment by akihikokuroda** [comment]
I see 2 executions in the log 

2020/02/13 16:08:08
and 
2020/02/13 16:09:35 

one wit "kabanero-pipeline" and the other with "kabanero-operator"?

**Comment by karimbzu** [comment]
@akihikokuroda yes initially i tried with "kabanero-operator" and build failed, and then "kabanero-pipeline" with the same result.

**Comment by akihikokuroda** [comment]
Let's focus of the latest execution.  Can I see the log of the tasks in the pipelinerun?

**Comment by karimbzu** [comment]
The succeeded one was build using "pipeline" service account  ::
[root@bastion 0.5]# tkn pipelinerun list
NAME                                                  STARTED          DURATION     STATUS
java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp   8 minutes ago    38 seconds   Failed
java-microprofile-build-deploy-pl-run-kjhrc           13 minutes ago   46 seconds   Failed
java-microprofile-build-deploy-pl-run-hqptm           35 minutes ago   15 minutes   Succeeded
java-microprofile-build-deploy-pl-run-lwx4r           46 minutes ago   2 minutes    Failed
[root@bastion 0.5]# tkn pipelinerun logs java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp
task build-deploy-task has failed: "step-build" exited with code 1 (image: "docker.io/appsody/appsody-buildah@sha256:6378803c47562d495b9e766bb9cd574d0d5dc58a0a924ee0581f93d30e9a74c5"); for logs run: kubectl -n kabanero logs java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp-build-8dmg8-pod-e4f479 -c step-build
TaskRun java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp-build-8dmg8 has failed
![image](https://user-images.githubusercontent.com/34478122/74458689-0ce3c900-4ec5-11ea-9a06-ab18b401444b.png)

**Comment by akihikokuroda** [comment]
@kvijai82 I remember I saw this error before.  This is an issue in the task or  Appsody.

**Comment by karimbzu** [comment]
[root@bastion 0.5]# tkn taskrun list
NAME                                                              STARTED          DURATION     STATUS
java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp-build-8dmg8   19 minutes ago   38 seconds   Failed
java-microprofile-build-deploy-pl-run-kjhrc-build-deploy--67jk4   24 minutes ago   46 seconds   Failed
java-microprofile-build-deploy-pl-run-hqptm-image-scan-ta-pl5tb   33 minutes ago   2 minutes    Succeeded
java-microprofile-build-deploy-pl-run-hqptm-build-deploy--kkw4f   46 minutes ago   12 minutes   Succeeded
java-microprofile-build-deploy-pl-run-lwx4r-build-deploy--755qx   57 minutes ago   2 minutes    Failed
[root@bastion 0.5]# tkn taskrun logs java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp-build-8dmg8
Error: task build-deploy-task has failed: "step-build" exited with code 1 (image: "docker.io/appsody/appsody-buildah@sha256:6378803c47562d495b9e766bb9cd574d0d5dc58a0a924ee0581f93d30e9a74c5"); for logs run: kubectl -n kabanero logs java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp-build-8dmg8-pod-e4f479 -c step-build

**Comment by akihikokuroda** [comment]
Would you do 
```
kubectl -n kabanero logs java-microprofile-build-deploy-pl-run-kjhrc-r-8smcp-build-8dmg8-pod-e4f479 -c step-build
```
?

**Comment by karimbzu** [comment]
![image](https://user-images.githubusercontent.com/34478122/74459698-bd060180-4ec6-11ea-8eb3-87ac96bdc0c9.png)

**Comment by karimbzu** [comment]
![image](https://user-images.githubusercontent.com/34478122/74459804-e030b100-4ec6-11ea-9f50-60892a7740ed.png)
Complete output

**Comment by akihikokuroda** [comment]
@kvijai82 I'm not familiar with the Appsody / kabanero stack but this looks suspicious.
```
Checking for configmap value....  DEFAULT_COLLECTION_IMAGE_URL=
DEFAUTL_COLLECTION_IMAGE_REGISTRY_URL= given as input by user is empty, so using the registry=docker.io for collection image download
```

**Comment by karimbzu** [comment]
I think i found the issue. Knative eventing is not successfully initialized in kabanero 
![image](https://user-images.githubusercontent.com/34478122/74512319-fd0aca00-4f42-11ea-97df-29e3922b0f3f.png)
Now, I am going to reinstall kabanaro from scratch after removing completely.

**Comment by kvijai82** [comment]
Oh interesting. Not sure why that happened.  Keep us posted on how things go with the new install.  Thanks!

**Comment by marikaj123** [comment]
@karimbzu - Can the issue be closed or the problem still exists?

**Comment by karimbzu** [comment]
The issue is still there, we are running two OCP 4.2 clusters simultaneously, one for staging and one for production. The pipelines in production cluster are able to use "kabanero-pipeline" service account whereas other cluster showing the same error as described. However, "pipeline" service account is working in both clusters but this causes another issue which is we cannot deploy pipeline apps on namespaces other than "kabanero".

**Comment by stephenkinder** [comment]
@karimbzu @marikaj123 @kvijai82 It is unclear to me at the moment what exactly we are blocked on.  Does the description of this issue still match the problem in the second cluster?

**Comment by kvijai82** [comment]
@karimbzu did you reinstall the staging cluster to get around the kNative issues you mentioned earlier?

**Comment by karimbzu** [comment]
@stephenkinder @kvijai82 @marikaj123, knative issue was resolved after reinstalling the whole kabanero 0.5. However, the previous issue is the same 
![image](https://user-images.githubusercontent.com/34478122/75304866-50541500-587f-11ea-8315-c33a54d4f7f2.png)

**Comment by kvijai82** [comment]
@karimbzu could you please run the must gather script on both your working and failing cluster and upload the logs?  There is also a new 0.6.0 release published if you want to give that a shot.  There are lot of changes in that release.  https://github.com/kabanero-io/kabanero-operator/releases/tag/0.6.0

**Comment by marikaj123** [comment]
@karimbzu - Please view the above comment from Vijai in order to provide you with better assistance.
